orca-interlink/src/main/java/com/netflix/spinnaker/orca/interlink/aws/InterlinkAmazonMessageHandler.java

```diff
@@ -59,14 +61,19 @@ public void handleMessage(Message message) {
 
   @VisibleForTesting
   void handleInternal(InterlinkEvent event) {
-    if (executionRepository.handlesPartition(event.getPartition())) {
-      event.applyTo(executionOperator);
-    } else {
-      log.debug(
-          "Execution repository with local partition {} does not handle this event's partition {}, "
-              + "so it will not be applied",
-          executionRepository.getPartition(),
-          event.getPartition());
+    try {
+      MDC.put(Header.EXECUTION_ID.getHeader(), event.getExecutionId());
```

Comment:
❤️ 
maybe also put x-spinnaker-user in here

Reply:
we don't have the user on every interlink event yet, but we probably will have to have it to handle manual stages later on